test/JDBC/input/key_column_usage-vu-prepare.sql

```diff
@@ -0,0 +1,23 @@
+use master;
```

Comment:
We should also create objects for example, view, function, procedure, etc dependent on view you are implementing and test them.

Reply:
Updated in latest commit.